service/resource/endpoint/resource.go

```diff
@@ -20,18 +21,62 @@ type Config struct {
 	Logger    micrologger.Logger
 }
 
+type Resource struct {
+	k8sClient kubernetes.Interface
+	logger    micrologger.Logger
+}
+
+func (r *Resource) Name() string {
```

Comment:
This is ok but we have the constructor after the type. There is nothing new here. Lets just have it like in all other projects. 

```
imports
constants
vars
config
type
constructor
method
function
```

```
public
private
```

```
dependencies
internals
settings
```

```
a
b
c
...
z
```

Reply:
Done.